build/scripts/build-utils.ps1

```diff
@@ -278,14 +275,29 @@ function Get-MSBuildDir([switch]$xcopy = $false) {
 # meets our minimal requirements for the Roslyn repo.
 function Get-VisualStudioDirAndId() {
     $vswhere = Join-Path (Ensure-BasicTool "vswhere") "tools\vswhere.exe"
-    $output = & $vswhere -requires Microsoft.Component.MSBuild -format json | Out-String
-    if (-not $?) {
-        throw "Could not locate a valid Visual Studio"
+    $output = Exec-Command $vswhere "-requires Microsoft.Component.MSBuild -format json" | Out-String
+    $j = ConvertFrom-Json $output
+    foreach ($obj in $j) { 
+
+        # Need to be using at least Visual Studio 15.2 in order to have the appropriate
+        # set of SDK fixes. Parsing the installationName is the only place where this is 
+        # recorded in that form.
+        $name = $obj.installationName
+        if ($name -match "VisualStudio(Preview)?/([\d.]+)(\+|-).*") { 
+            $minVersion = New-Object System.Version "15.1.0"
```

Comment:
Comment says `at least Visual Studio 15.*2*`, this says `15.*1*.0`. Which one is it?